modules/sno-about-sno-cluster-worker-nodes.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+:_module-type: CONCEPT
+[id="sno-about-sno-cluster-worker-nodes_{context}"]
+= About worker nodes on single-node OpenShift clusters
+
+Worker nodes for single-node OpenShift clusters provide additional resources for your cluster. However, because there is only a single control plane node, ingress traffic is routed differently compared to standard cluster configurations with three control plane nodes.
+
+In a default single-node cluster configuration, all ingress traffic is routed through the control plane node by default. One of the benefits of installing single-node clusters is the simplicity of not having to deal with load-balancing and virtual IPs. Not having a load balancer installed by default increases cost savings and simplifies cluster deployment and configuration.
```

Comment:
>In a default single-node cluster configuration, all ingress traffic is routed through the control plane node by default.

This sentence feels a bit off. When you have just one node, it's trivial that all ingress traffic goes through it, because it has nowhere else to go.

When there is more than one node involved, it's less trivial to realize ingress traffic must flow the original SNO node, so that has to be mentioned